content/raceday.inc

```diff
@@ -7,20 +7,24 @@
       <a class="nav-link" href="/raceday/heats">Heat Schedule</a>
     </li>
     <li class="nav-item">
-      <a class="nav-link" href="/history">Leaderboard</a>
+      <a class="nav-link" href="/history/rosters/2022">Rosters</a>
```

Comment:
I understand why you're fixing this to 2022, but it becomes one of those things that needs to be updated every year.

I guess it could be worse, since you also have to update the wordpress verison of the /raceday page in the same way, but it also might not be super obvious in the future that you need to fix it in two places.

Not going to /history though is clearly the right thing.

Possibilities:
- SQL query to find the most recent "valid" roster year, if it isn't date("Y"), then grey this item out and make it unclickable or something.
- Just go to /history/rosters and let the logic on that page sort it out, assume we don't launch until april 1
- Ignore the problem and wait until we forget about this next year :)

Reply:
Great point. Worse is that this nav is replicated in WordPress which makes option1 definitely not possible.

Would it be horrible to change the /history/rosters logic to flip to the current year in March? Now that we have the "hold your horses and check back later" bit?

Reply:
(I made this update, let me know if you hate it hahah)

Reply:
Seems fine